strings/index.js

```diff
@@ -56,7 +62,11 @@ export const characterExists = (string, char) => {
   character in a string
 */
 export const whereIsCharacter = (string, char) => {
-  return string.indexOf(char);
+  if (string.indexOf(char) === -1) {
+    return false
+  } else {
+    return string.indexOf(char);
+  }
```

Comment:
Once you're returned out of a function the code stops executing, so you actually don't need the else.

```js
if (x) {
  return true;
}
return false
```

Also, you've repeated a tiny bit of code with the string.indexOf() check, you could set that into a variable first, then do a check if it's -1 (At which point you return false) then just return the actual value

nice and cleeann